config_parse.go

```diff
@@ -28,6 +30,11 @@ func ReadProxyConfig(path string) (c ProxyConfig, err error) {
 		return
 	}
 
+	err = envconfig.Process("veneur", &c)
+	if err != nil {
+		log.Fatal(err.Error())
```

Comment:
Derp, yes. I didn't realize i was in a place where I could return an `Error` and the caller could decide. Thank you!